src/uct/api/uct_def.h

```diff
@@ -111,6 +117,13 @@ typedef int                      uct_worker_cb_id_t;
     |<---- stride ----->|<---- stride ----->|
    @endverbatim
  *
+ * In order to describe an interleaved memory layout, each structure contains
```

Comment:
I don't see how, because it's ambiguous. For each letter you need to store both count (how many items), length (per-itme) and pattern (ilv_ratio). For example, how would you encode the difference between ABABABAB and AABBAABB ?